src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml

```diff
@@ -203,7 +203,11 @@ Resources:
           - Id: CleanupOldVersions
             Status: Enabled
             NoncurrentVersionExpiration:
-              NoncurrentDays: 365
+              NoncurrentDays: 30
```

Comment:
please double check this update @philasmar

Reply:
Changes are valid